examples/advanced/random_forest/README.md

```diff
@@ -34,10 +34,9 @@ No further training will be performed, `num_boost_round` should be 1 to align wi
 ## HIGGS Data Preparation
 ### Download and Store Data
 To run the examples, we first download the dataset from the HIGGS link above, which is a single `.csv` file.
-By default, we assume the dataset is downloaded, uncompressed, and stored in `~/dataset/HIGGS.csv`.
+By default, we assume the dataset is downloaded, uncompressed, and stored in `DATASET_ROOT/HIGGS.csv`.
```

Comment:
Not related to this PR, but in later PR we need to think of how to combine this with the XGBoost tree-based example instead of duplicating them